Makefile

```diff
@@ -125,6 +125,8 @@ defn_files:=$(ocaml_files) $(java_files) $(node_files)
 
 defn: $(defn_files)
 
```

Comment:
No extra space. Also add the corresponding targets `java-defn`, `ocaml-defn`, `node-defn` please.